muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/reading-panel.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
```

Comment:
If this file was renamed it means that all the other files who used the previous application-panel view are to be broken, they need to be renamed as well.

Reply:
Nothing was renamed, reading-panel.tsx was created from application-panel.tsx. Although reading-panel name might need some evaluating as it might be too generic even though that's the idea.

Reply:
Ok then no problem I'd guess so, but once I start cleaning up one of them has to go if they represent the same concept, nevertheless, it's good for a pull now.